wsclient/src/test/java/org/hpccsystems/ws/client/platform/test/PlatformTester.java

```diff
@@ -4,35 +4,25 @@
 import java.io.File;
 import java.io.FileWriter;
 import java.io.IOException;
-import java.io.Writer;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Set;
 import java.util.StringTokenizer;
 
 import org.hpccsystems.ws.client.HPCCFileSprayClient;
 import org.hpccsystems.ws.client.HPCCWsClient;
 import org.hpccsystems.ws.client.HPCCWsDFUClient;
 import org.hpccsystems.ws.client.extended.HPCCWsAttributesClient;
 import org.hpccsystems.ws.client.extended.HPCCWsSQLClient;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05allverinclusive.ExecuteSQLResponse;
-import org.hpccsystems.ws.client.gen.filespray.v1_15allverinclusive.PhysicalFileStruct;
-import org.hpccsystems.ws.client.gen.wsdfu.v1_38allverinclusive.DFUDataColumn;
-import org.hpccsystems.ws.client.gen.wsworkunits.v1_71allverinclusive.WUPublishWorkunitResponse;
-import org.hpccsystems.ws.client.platform.DFUDataColumnInfo;
+import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.ExecuteSQLResponse;
+import org.hpccsystems.ws.client.gen.filespray.v1_15.PhysicalFileStruct;
+import org.hpccsystems.ws.client.gen.wsdfu.v1_39.DFUDataColumn;
 import org.hpccsystems.ws.client.platform.DFUFileDetailInfo;
-import org.hpccsystems.ws.client.platform.DFUFilePartsOnClusterInfo;
 import org.hpccsystems.ws.client.platform.DFUFilePartInfo;
-import org.hpccsystems.ws.client.platform.DFURecordDefInfo;
+import org.hpccsystems.ws.client.platform.DFUFilePartsOnClusterInfo;
 import org.hpccsystems.ws.client.platform.DropZone;
-import org.hpccsystems.ws.client.platform.EclRecordInfo;
+import org.hpccsystems.ws.client.platform.LogicalFile;
 import org.hpccsystems.ws.client.platform.Platform;
 import org.hpccsystems.ws.client.platform.Version;
-import org.hpccsystems.ws.client.platform.WorkunitInfo;
 import org.hpccsystems.ws.client.platform.test.data.Accounts;
 import org.hpccsystems.ws.client.platform.test.data.Persons;
-import org.hpccsystems.ws.client.utils.Connection;
-import org.hpccsystems.ws.client.utils.FileFormat;
```

Comment:
I'm assuming a lot of these imports were removed because they were marked as unused by eclipse?

Reply:
yep, there was an automatic "organize imports"